Tests/MapboxNavigationTests/MapViewTests.swift

```diff
@@ -251,7 +251,7 @@ class MapViewTests: TestCase {
         
         // Simplified Chinese is broken: https://github.com/mapbox/mapbox-maps-ios/issues/652
         mapView.localizeLabels(into: Locale(identifier: "zh-Hans-CN"))
-        assert(roadLabelProperty: "name_en", placeLabelProperty: "name_zh")
+        assert(roadLabelProperty: "name_en", placeLabelProperty: "name_zh-Hans")
```

Comment:
@1ec5, I guess it's expected result now based on https://github.com/mapbox/mapbox-maps-ios/pull/1602?